paper/paper.md

```diff
@@ -209,137 +208,50 @@ With this information, a Bayesian model can be specified in Turing.jl [@Turing.j
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+When sampling the posterior the log probability density function `logpdf` implemented in `JointModels.jl` is called for a distribution given specific parameters. The numerical calculation of the likelihood is then used in the sampling process. Sampling with the `Turing.Inference.NUTS` algorithm 2000 posterior samples using 1000 burn-in results in posterior statistic:
 
+```
+parameters        mean        std      mcse       rhat  |  ture_parameters    
+    Symbol     Float64    Float64   Float64    Float64  |       
+
+    μ_BSLD     61.3971     4.4043    0.2737     1.0021  |        60
+       μ_d      0.0058     0.0014    0.0001     0.9998  |         0.0055 
+       μ_g      0.0018     0.0002    0.0000     0.9998  |         0.0015
+       μ_φ      0.1729     0.0591    0.0049     1.0001  |         0.2
+         σ      0.1767     0.0057    0.0001     1.0008  |         0.18
+         λ   1491.2720   307.5059    6.4314     0.0008  |      1450
+         γ      0.0103     0.0012    0.0000     1.0005  |         0.01
+```
+Notice that the link coefficient $\gamma$ was sampled around the true value $0.01$ with a narrow variance. The survival and population parameters are well represented by the posterior samples indicated by the $\hat r$ value, which is close to one and the relative closeness to the true parameter.
 
 
+Additionally, `JointModels.jl` implements the generation of random samples of a joint distribution which enables `Turing.jl` to sample a joint distribution. This allows the creation of posterior predictive checks, simulations, or individual predictions \autoref{fig:ind_pred}, which are a major step in a Bayesian workflow when validating a model [@BayesianWorkflow]. 
 
+![The figure showcases individual posterior prediction for the mixed effects longitudinal sub-model along side the longitudinal observations. The figure also shows the individual survival predictions conditioned on survival until the last measurement based on the joint survival distribution. The light-colored ribbons represent the 95% quantile of the posterior predictions while the line represents the median. To create the posterior predictions the above `Turing.jl` model was used. \label{fig:ind_pred}](individual_prediction.svg){ width=80% }
 
 
 
 
 
 
+# Acknowledgements
 
+Professor Peter Bühlmann from ETH Zürich enabled the development and subsequently open source project. The initial development was made at Hoffmann-La Roche Ltd.
```

Comment:
the first sentence sounds not quite right, maybe start with Roche (chronological) and then in the ETH sentence "enabled the further development."? I don't think P.B. needed to approve the open sourcing? (that was more Roche?) so I would not mention that

Reply:
@danielinteractive what do you think about:

>This software was initially developed at Hoffmann-La Roche Ltd. before transitioning into an open source project. Professor Peter Bühlmann from ETH Zürich facilitated the development and open source project.

Its hard to have a truly chronological account, since he was responsible for the first contact and trust.

Reply:
sounds good like this!